app/pages/sections/additional-services/routes.js

```diff
@@ -67,5 +72,54 @@ export const additionalServicesRoutes = (authProvider, addContext, sessionManage
     return res.render('pages/sections/additional-services/order-item/template.njk', addContext({ context, user: req.user, csrfToken: req.csrfToken() }));
   }));
 
+  router.post('/:orderItemId', authProvider.authorise({ claim: 'ordering' }), withCatch(logger, authProvider, async (req, res) => {
+    const { orderId, orderItemId } = req.params;
+    const validationErrors = [];
+
+    const accessToken = extractAccessToken({ req, tokenType: 'access' });
+    const pageData = sessionManager.getFromSession({ req, key: 'orderItemPageData' });
+
+    const errors = validateOrderItemForm({
+      orderItemType: 'additional-services',
+      data: req.body,
+      selectedPrice: pageData.selectedPrice,
+    });
+    validationErrors.push(...errors);
+
+    if (validationErrors.length === 0) {
+      const apiResponse = await saveOrderItem({
+        accessToken,
+        orderId,
+        orderItemId,
+        serviceRecipientId: pageData.serviceRecipientId,
+        serviceRecipientName: pageData.serviceRecipientName,
+        itemId: pageData.itemId,
+        itemName: pageData.itemName,
+        selectedPrice: pageData.selectedPrice,
+        formData: req.body,
+      });
+
+      if (apiResponse.success) {
+        logger.info('redirecting additional-services main page');
```

Comment:
'redirecting additional-services main page' -> 'redirecting to the additional-services main page' ?